samples/wifi/shell/src/wifi_main.c

```diff
@@ -15,4 +50,9 @@ void main(void)
 			       NRF_CLOCK_HFCLK_DIV_1);
 #endif
 	printk("Starting %s with CPU frequency: %d MHz\n", CONFIG_BOARD, SystemCoreClock/MHZ(1));
+
+	net_if_flag_set(net_if_get_default(), NET_IF_NO_AUTO_START);
```

Comment:
Also, I would recommend getting the `wlan0` handle than using default interface as we have multiple interface usecases.